rivet.sh

```diff
@@ -179,16 +164,18 @@ cat << EOF > source3rd
 test -f \$prefix/etc/rivet_3rdparty.sh && source \$prefix/etc/rivet_3rdparty.sh
 EOF
 
+# Make back-up of original for debugging - disable execute bit
+cp $INSTALLROOT/bin/rivet-config $INSTALLROOT/bin/rivet-config.orig
+chmod 644 $INSTALLROOT/bin/rivet-config.orig
 # Modify rivet-config script to use environment from rivet_3rdparty.sh
 cat $INSTALLROOT/bin/rivet-config | sed -e "$SED_EXPR" > $INSTALLROOT/bin/rivet-config.0
 csplit $INSTALLROOT/bin/rivet-config.0 '/^datarootdir=/+1'
 cat xx00 source3rd xx01 >  $INSTALLROOT/bin/rivet-config
 chmod 0755 $INSTALLROOT/bin/rivet-config
-# Show the script on standard output - For debugging
-# echo "=== Configuration script is ===="
-# cat $INSTALLROOT/bin/rivet-config 
-# echo "=== End of configuration script ===="
 
+# Make back-up of original for debugging - disable execute bit
+cp $INSTALLROOT/bin/rivet-build $INSTALLROOT/bin/rivet-build.orig
```

Comment:
I think we will remove these "installs" when we see that all is well, so I think using `cp` for now is OK.  Otherwise, I agree that `install` is a better option.